gitlab/v4/objects/registry_protection_rules.py

```diff
@@ -0,0 +1,35 @@
+from gitlab.base import RESTManager, RESTObject
+from gitlab.mixins import CreateMixin, ListMixin, SaveMixin, UpdateMethod, UpdateMixin
+from gitlab.types import RequiredOptional
+
+__all__ = [
+    "ProjectRegistryProtectionRule",
+    "ProjectRegistryProtectionRuleManager",
+]
+
+
+class ProjectRegistryProtectionRule(SaveMixin, RESTObject):
+    _repr_attr = "repository_path_pattern"
+
+
+class ProjectRegistryProtectionRuleManager(
+    ListMixin, CreateMixin, UpdateMixin, RESTManager
+):
+    _path = "/projects/{project_id}/registry/protection/rules"
+    _obj_cls = ProjectRegistryProtectionRule
+    _from_parent_attrs = {"project_id": "id"}
+    _create_attrs = RequiredOptional(
+        required=("repository_path_pattern",),
+        optional=(
+            "minimum_access_level_for_push",
+            "minimum_access_level_for_delete",
+        ),
+    )
+    _create_attrs = RequiredOptional(
```

Comment:
These are both `_create_attrs`

Reply:
Ugh! Sorry, was a quick one. Fixed in https://github.com/python-gitlab/python-gitlab/compare/e0fa822b1beb45e3dc9a164b62824d3f9d544158..90a6cdf00af38cdab07988c3f276062966e758bc.